app/src/components/existing-request/nr-approved-gray-box.vue

```diff
@@ -337,7 +342,14 @@ export default class NrApprovedGrayBox extends Mixins(CommonMixin) {
       this.isApprovedOrConsentUnRequired
     )
   }
+
+  /** check if business is in Lear and set store value of isLearBusiness flag */
+  async checkBusiness (): Promise<void> {
+    await this.checkBusinessInLear(this.getNr?.corpNum)
```

Comment:
Why not just await this in mounted() (and make that async to match)?

Reply:
Yes. I think that is better.

Reply:
Beauty! Simple is best. Resolving.